shared/src/business/utilities/copyPagesFromPdf.js

```diff
@@ -0,0 +1,12 @@
+const copyPagesFromPdf = async ({ copyFrom, copyInto }) => {
+  let pagesToCopy = await copyInto.copyPages(
+    copyFrom,
+    copyFrom.getPageIndices(),
+  );
+
+  pagesToCopy.forEach(page => {
+    copyInto.addPage(page);
```

Comment:
It would appear this always appends the copied pages to the end of the target PDF. Maybe the method name (or JSDoc) should reflect this limitation?

Reply:
Addressed. JSDOC created and renamed.